packages/loot-core/migrations/1722804019000_create_dashboard_table.js

```diff
@@ -17,7 +17,8 @@ export default async function runMigration(db) {
       INSERT INTO dashboard (id, type, width, height, x, y)
       VALUES
         ('${uuidv4()}','net-worth-card', 8, 2, 0, 0),
-        ('${uuidv4()}', 'cash-flow-card', 4, 2, 8, 0);
+        ('${uuidv4()}', 'cash-flow-card', 4, 2, 8, 0),
+        ('${uuidv4()}', 'spending-card', 4, 2, 0, 2);
```

Comment:
shouldn't this be a new migration and not an old one that has already been applied?

Reply:
Usually yes, but I don't want to add to any existing dbs. This is only applying to brand new instances. A new migration would inadvertently add a spending-card to everyone's report dashboard. I only want the spending report to show up as one of the default graphs for new installs and do nothing to existing ones. Hence the change to an existing migration. 

I know editing existing migrations is generally not a good idea, but this is the best way I see to accomplish the end goal without causing any disruption to everyone already using the reports dashboard. I'm happy to be wrong, let me know if there's a better way to do this!